bot/exts/info/pypi.py

```diff
@@ -63,6 +72,10 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
                     else:
                         embed.description = "No summary provided."
 
+                    upload_time = _get_latest_distribution_timestamp(response_json)
+                    if upload_time:
+                        embed.timestamp = upload_time
```

Comment:
Getting the conversation going again, perhaps something simple like this would work:

```suggestion
                        embed.timestamp = f"Uploaded on {upload_time}"
```

Reply:
Yes, I like this better, as it doesn't imply the embed's timestamp but rather the version's.